Admin/AbstractAdmin.php

```diff
@@ -584,7 +584,21 @@ public function getDataSourceIterator()
         $datagrid = $this->getDatagrid();
         $datagrid->buildPager();
 
-        return $this->getModelManager()->getDataSourceIterator($datagrid, $this->getExportFields());
+        $fields = array();
+
+        foreach ($this->getExportFields() as $key => $field) {
+            $label = $this->getTranslationLabel($field, 'export', 'label');
+            $transLabel = $this->trans($label);
+
+            // No translation key exists
+            if ($transLabel == $label) {
+                $fields[$key] = $field;
```

Comment:
Will this be removed on next major? Because it will trigger the "missing translation" sf mechanism, right?